File: teledonCS/model/Volunteer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace teledonCS.model
{
    class Volunteer
    {
        public Volunteer(int id, string firstName, string lastName)
        {
            this.id = id;
            this.firstName = firstName;
            this.lastName = lastName;
        }

        private int id { get; set; }
        private string firstName { get; set; }
        private string lastName { get; set; }

        public override string ToString()
        {
            return base.ToString();
        }
    }
}

File: teledon/src/main/java/model/Volunteer.java
package model;

public class Volunteer implements Identifiable<Integer>{
    private int id;
    private String firstName;
    private String lastName;

    public Volunteer(int id, String firstName, String lastName) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
    }

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer id) {
        this.id = id;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    @Override
    public String toString() {
        return "Volunteer{" +
                "id=" + id +
                ", firstName='" + firstName + '\'' +
                ", lastName='" + lastName + '\'' +
                '}';
    }
}
